### PR TITLE
Checkbox toggle shortcut

### PR DESCRIPTION
**Pull request by mikailcf**
- [x] I understand that contributing to this repository will require me to agree with the [CLA](https://github.com/anyproto/.github/blob/main/docs/CLA.md)

---

### What
This PR adds a new shortcut for checking/unchecking checkboxes and suggests a change to the shortcut for expanding/collapsing toggles for both to be the same.

### Why
It's very practical to be able to mark checkbox items in a list when you can do that directly from the keyboard in a text heavy application.

The suggestion for the toggle shortcut is to keep both equal, since they both have similar effects, and with easier access. Since `cmd + enter` wasn't being currently used in Anytype and other similar apps in the market use that combination, I thought it would make sense.

### What type of PR is this? (check all applicable)

- [x] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI

### Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

### Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 [tech-docs](https://github.com/anyproto/tech-docs)
- [ ] 🙅 no documentation needed

### CODE REVIEW

File: src/json/text.json [hunk 1]
```diff
@@ -1129,7 +1129,7 @@
 	"popupShortcutNavigationMenu4": "Go to the right side of navigation. Link to page",
 	"popupShortcutNavigationMenu5": "Select option",
 	"popupShortcutNavigationPage": "Editor navigation",
-	"popupShortcutNavigationPage1": "Expand / Collapse Toggle",
+	"popupShortcutNavigationPage1": "Expand / Collapse Toggle; Check / Uncheck Checkbox",
```
Comment (ra3orblade) [inline src/json/text.json:1132]: This is not correct way to add strings, it's 2 different strings one of which is already translated into 20 languages.

File: src/ts/component/editor/page.tsx [hunk 2]
```diff
@@ -793,10 +793,18 @@ const EditorPage = observer(class EditorPage extends React.Component<Props, Stat
 			});
 
 			if (block.isTextToggle()) {
-				keyboard.shortcut(`${cmd}+shift+t`, e, () => {
```
Comment (ra3orblade) [inline src/ts/component/editor/page.tsx:796]: Changing the shortcut itself should be more of a feature request, I can't allow this without a product review.

File: src/ts/component/editor/page.tsx [hunk 3]
```diff
@@ -793,10 +793,18 @@ const EditorPage = observer(class EditorPage extends React.Component<Props, Stat
...
 				});
 			};
+
+			if (block.isTextCheckbox()) {
+				keyboard.shortcut(`${cmd}+enter`, e, () => {
+					UtilData.blockSetText(rootId, block.id, text, marks, true, () => {
+						C.BlockTextSetChecked(rootId, block.id, !block.content.checked);
+					});
+				});
+			};
```
Comment (ra3orblade) [inline src/ts/component/editor/page.tsx:807]: This code should be combined with the code above, its only different in Api command

**Review by ra3orblade — CHANGES_REQUESTED** [review]

File: src/json/text.json (hunk 1 above)
Comment (mikailcf) [reply src/json/text.json:1132]: Ok, there aren't any kind of guides for that, so I didn't know. Could you tell me what's the correct way of adding a new string and translations for it then?

File: src/ts/component/editor/page.tsx (hunk 2 above)
Comment (mikailcf) [reply src/ts/component/editor/page.tsx:796]: I thought that might be the case. I'll revert back.

File: src/json/text.json (hunk 1 above)
Comment (ra3orblade) [reply src/json/text.json:1132]: There is no actual guide, but if you change something it needs to be translated again and it creates less friction when it's a new line, especially with such a case when new line is added to the same place like here.

Now I think you can leave it like this, does not matter too much.

File: src/ts/component/editor/page.tsx (hunk 3 above)
Comment (mikailcf) [reply src/ts/component/editor/page.tsx:807]: I've pushed new code separating the shortcuts.

File: src/json/text.json (hunk 1 above)
Comment (mikailcf) [reply src/json/text.json]: I see. I've push new code adding a new string and leaving the old as-is. See if it helps.

**Review by ra3orblade — APPROVED** [review]

**Review by ra3orblade — APPROVED** [review]

**Comment by fuksman** [comment]
@any contributor @mikailcf code

**Pull request merged**